### DISCUSSION TITLE
HTTP proxy for loading pods?

### DISCUSSION THREAD

**Discussion by sunng87** [babashka/babashka#1497] [Q&A] [answered]
My network seems to require http proxy to access the pod registry. I wonder if proxy configuration works for babashka, for exmaple, the `http_proxy` env or `-Dhttp.httpProxy` from java.

**Comment by borkdude** [comment, marked as answer]
@sunng87 Hi! The code where the pod is downloaded is here:

https://github.com/babashka/pods/blob/75c2216649bf5caf3ae41e7b8fc202e45b090ea9/src/babashka/pods/impl/resolver.clj#L106-L115

I suspect that `bb -Dhttps.proxyHost=...` should work.

E.g. I see that 

```
bb -Dhttps.proxyHost=foobar -e '(slurp "https://clojure.org")'
```

fails, because of the invalid proxy host. You. could use the `BABASHKA_PRELOADS` environment variable to programmatically set the system property as well. Let me know if this works for you or not.

**Comment by sunng87** [comment]
Thank you very much! That worked for me.
